ksqldb-functional-tests/src/test/resources/query-validation-tests/row-offset-and-partition.json

```diff
@@ -119,6 +150,622 @@
         "type": "io.confluent.ksql.util.KsqlStatementException",
         "message": "'ROWPARTITION' is a reserved column name. You cannot use it as a name for a column."
       }
+    },
+    {
+      "comments": "Should be removed after rowpartition.rowoffset.enabled feature flag is removed",
+      "name": "test ROWOFFSET and ROWPARTITION source user columns with pseudocolumn data types- ROWOFFSET/ROWPARTITION enabled",
+      "properties": {
+        "ksql.rowpartition.rowoffset.enabled": true
+      },
+      "statements": [
+        "CREATE STREAM input (id INT KEY, ROWPARTITION INTEGER, ROWOFFSET BIGINT) WITH (kafka_topic='test_topic', value_format='JSON');"
+      ],
+      "expectedException": {
+        "type": "io.confluent.ksql.util.KsqlStatementException",
+        "message": "'ROWPARTITION' is a reserved column name. You cannot use it as a name for a column."
+      }
+    },
+    {
+      "name": "INSERT INTO ROWPARTITION should fail",
+      "properties": {
+        "ksql.rowpartition.rowoffset.enabled": true
+      },
+      "statements": [
+        "CREATE STREAM input (id INT KEY, val INT) WITH (kafka_topic='test_topic', value_format='JSON');",
+        "INSERT INTO input (id, val, rowpartition) VALUES (1, 2, 3);"
+      ],
+      "expectedException": {
+        "type": "io.confluent.ksql.util.KsqlStatementException",
+        "message": "Failed to insert values into 'INPUT'. Inserting into column `ROWPARTITION` is not allowed."
+      }
+    },
+    {
+      "name": "INSERT INTO ROWOFFSET should fail",
+      "properties": {
+        "ksql.rowpartition.rowoffset.enabled": true
+      },
+      "statements": [
+        "CREATE STREAM input (id INT KEY, val INT) WITH (kafka_topic='test_topic', value_format='JSON');",
+        "INSERT INTO input (id, val, rowoffset) VALUES (1, 2, 3);"
```

Comment:
This is not the right test framework to test this. Only RQTT supports INSERT VALUES. See insert-values.json.

Reply:
I'm confused- I thought we made the distinction between INSERT INTO and INSERT VALUES? https://github.com/confluentinc/ksql/pull/8121#discussion_r707605438

Reply:
The test you've added here is for INSERT VALUES, not INSERT INTO.
https://docs.ksqldb.io/en/latest/developer-guide/ksqldb-reference/insert-values/
https://docs.ksqldb.io/en/latest/developer-guide/ksqldb-reference/insert-into/